packages/models/src/crypto-assets/crypto-asset-balance.model.ts

```diff
@@ -0,0 +1,41 @@
+import { Money } from '../money.model';
+
+/** Balance types:
+ * availableBalance: totalBalance after filtering out outboundBalance, protectedBalance, and uneconomicalBalance
+ * availableUnlockedBalance; availableBalance minus lockedBalance
+ * inboundBalance - balance of pending receipt into account given pending transactions
+ * lockedBalance - totalBalance minus total amount locked by contracts
+ * outboundBalance - balance of pending delivery from account given pending transactions
+ * pendingBalance - totalBalance plus inboundBalance minus outboundBalance
+ * protectedBalance - balance of UTXOs with collectibles (if bitcoin)
+ * totalBalance - balance as confirmed on chain
+ * uneconomicalBalance - balance across UTXOs with need for larger fee than principal by UTXO given standard rate (if bitcoin)
+ * unlockedBalance - totalBalance minus lockedBalance
+ */
+
+export interface CryptoAssetBalance {
+  readonly address: string;
+  readonly addressIndex: number;
+  readonly assetId: string; // Name or symbol
```

Comment:
Yep, will do, comment was left by mistake bc I'm not sure the `assetId` is necessary? I was just thinking it would be a good identifier to have with the balance to understand what asset info it relates to.